src/targz_sizes.c

```diff
@@ -120,8 +126,24 @@ int main(int argc, char** argv) {
         }
     }
 
+    unsigned int max_filename_blocks;
+    if ( LEVEL_IS_TEST( verbosity ) ) {
```

Comment:
I'm sorry that you are having to pick-up formatting inconsistencies.  Addressed in commit: 884ca7e05ad490a629e41738a69fe938cd74682a